### PR TITLE
DMEERX-794 Validate Mettles Rule Sets

### PR DESCRIPTION
**Pull request by yuliumitre**
This PR modifies CRD for make Mettles rule sets work. 

To test, use test-ehr and CDS-Libary prosrule branches.

### CODE REVIEW

File: resources/src/main/java/org/hl7/ShortNameMaps.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
   static {
     PAYOR_SHORT_NAME_TO_FULL_NAME.put("cms", "Centers for Medicare and Medicaid Services");
     PAYOR_SHORT_NAME_TO_FULL_NAME.put("uhc", "United Health Care");
+    PAYOR_SHORT_NAME_TO_FULL_NAME.put("medicare", "MEDICARE");
```
Comment (plarocque4) [inline resources/src/main/java/org/hl7/ShortNameMaps.java:19]: Is Medicare a different payer from CMS?

File: server/src/main/java/org/hl7/davinci/endpoint/controllers/FhirController.java [hunk 2]
```diff
@@ -76,7 +76,7 @@
       // If URL starts with this server's base url, pull out id and search by id
       if (url.startsWith(baseUrl)) {
         String valueSetId = url.split("ValueSet/")[1];
-        FileResource fileResource = fileStore.getFhirResourceById("R4", "valueset", "valueset/" + valueSetId, baseUrl);
+        FileResource fileResource = fileStore.getFhirResourceById("R4", "valueset", valueSetId, baseUrl);
```
Comment (plarocque4) [inline server/src/main/java/org/hl7/davinci/endpoint/controllers/FhirController.java:79]: You verified that this still works with all of our previous valuesets and rule sets?

File: resources/src/main/java/org/hl7/ShortNameMaps.java (hunk 1 above)
Comment (yuliumitre) [reply resources/src/main/java/org/hl7/ShortNameMaps.java:19]: I'm actually not sure.

File: server/src/main/java/org/hl7/davinci/endpoint/controllers/FhirController.java (hunk 2 above)
Comment (yuliumitre) [reply server/src/main/java/org/hl7/davinci/endpoint/controllers/FhirController.java:79]: Yes, as our RI actually doesn't use those locally defined value sets. There is only one defined - COPD (maybe for demo purpose?), but it is not used anywhere. This is a path mixup.

File: resources/src/main/java/org/hl7/ShortNameMaps.java (hunk 1 above)
Comment (lpdecelles) [reply resources/src/main/java/org/hl7/ShortNameMaps.java:19]: "Medicare Advantage" or MA is the plan that CMS offers. You might see Medicare Advantage commercials on TV. So we should remove the MEDICARE line.

Comment (lpdecelles) [reply resources/src/main/java/org/hl7/ShortNameMaps.java:19]: @yuliumitre said she will remove the "Medicare Advantage" line.

**Pull request merged**